Directory.Build.props

```diff
@@ -1,6 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
+
+  <PropertyGroup>
+    <!-- Don't need Microsoft.Common.props to import Directory.Build.props again -->
+    <ImportDirectoryBuildProps>false</ImportDirectoryBuildProps>
```

Comment:
Why is this necessary?

Reply:
Should I make the comment better here?

Reply:
The comment is definitely added confusion to me as I was expecting this to automatically be imported. Is there any way we can just let the SDK import these in those project types as well? Or are they too much work right now to convert to SDK?

Reply:
The way `Directory.Build.props|targets` files get automatically imported is by someone importing `Microsoft.Common.props|targets`.  If your project isn't an "SDK-style" project, then somewhere in your project, you need an explicit `<Import>` statement in order to get the `Microsoft.Common.props|targets` imported.

So in order to do that, we would either need to:
1. Make depproj and pkgproj (basically ALL our project files in corefx) be some sort of "SDK-style" projects. And in that "SDK", it would need to import `Microsoft.Common.props|targets`.
2. Explicitly `Import` a different `.props|.targets` set of files at the top and bottom of the .depproj and .pkgproj files.  Then that other set of `.props|.targets` files would import `Microsoft.Common.props|targets` at the right place, which would then import the `Directory.Build.props|targets` correctly.

I haven't tried (1) above, yet.  You said @joperezr was working on the pkgproj projects, so I didn't want to do any work there.  Also, it feels like we should try to make incremental progress here - get the .csproj's converted first, then worry about .depproj and .pkgproj.

Reply:
For now I'm fine if you could please update the comment to call out this is about dealing with the non-sdk projects.

Reply:
Done.